src/targets/github.ts

```diff
@@ -291,7 +282,7 @@ export class GitHubTarget extends BaseTarget {
     const name = basename(path);
 
     if (isDryRun()) {
```

Comment:
As I mentioned earlier, we should have _no_ `isDryRun()` calls inside "regular" code unless it is unavoidable.